airqo-platform/AirQo-api
# Github workflow cleanup

# Github workflow cleanup

**_WHAT DOES THIS PR DO?_**

- [x] Removes outdated workflows from github actions
- [x] Adds `kubectl apply -f` to all workflows
- [x] Transfers data-mgt microservice environment variables to a config map
- [x] Cleans up the data management microservice deployment file

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
None

**_HOW DO I TEST OUT THIS PR?_**
Successful merging of the PR should build and deploy all microservices without fail

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
All Endpoints of all microsevices 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

N/A

**_ARE THERE ANY RELATED PRs?_**

N/A
